### PR TITLE
feat(apple): Add initialScope option

### PR DESCRIPTION
**Pull request by philipphofmann**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Add intialScope option for the Apple SDK. Related PR https://github.com/getsentry/sentry-cocoa/pull/2982. Only merge after releasing Cocoa 8.7.0.

### CODE REVIEW

**Review by brustolin — APPROVED** [review]

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -423,6 +423,38 @@ Sentry.init({
 
 </ConfigKey>
 
+<ConfigKey name="initial-scope" supported={["apple"]}>
+
+This feature is available on [sentry-cocoa 8.7.0](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#870) and above.
```
Comment (shanamatthews) [inline src/platforms/common/configuration/options.mdx:428]:
```suggestion
```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -423,6 +423,38 @@ Sentry.init({
...
+
+[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
+    options.dsn = @"___PUBLIC_DSN___";
+    options.initialScope = ^(SentryScope *scope) {
+        scope setTagValue:@"my value" forKey:@"my-tag"];
+        return scope;
+    };
+}];
+```
+
```
Comment (shanamatthews) [inline src/platforms/common/configuration/options.mdx:455]:
```suggestion

_(New in [sentry-cocoa version 8.7.0](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#870))_
```

It looks like the convention is to style it this way.

Comment (shanamatthews) [reply src/platforms/common/configuration/options.mdx:455]: hmmmm idk if this is rendering correctly - please double check!

**Review by shanamatthews — APPROVED** [review]
Just one little tweak about where you place this info. I'm not sure if the markdown was being rendered correctly though, so please double-check for me!

**Pull request merged**